.eslintrc

```diff
@@ -1,7 +1,6 @@
 {
-  "parser": "babel-eslint",
```

Comment:
We don't need this anymore and projects in Skyscanner don't use this - alongside this `babel-eslint` is deprecated as well so removing it makes sense